libs/ledgerjs/packages/errors/src/index.ts

```diff
@@ -281,6 +281,7 @@ export const StatusCodes = {
   GP_AUTH_FAILED: 0x6300,
   LICENSING: 0x6f42,
   HALTED: 0x6faa,
+  LOCKED_DEVICE: 0x5515,
```

Comment:
What's the point of adding this here if we're hardcoding the 0x5515 instead of comparing against this?

Reply:
🙈 indeed

Well it is actually necessary as it is used just below in the `TransportStatusError` definition:
```TS
Object.keys(StatusCodes).find((k) => StatusCodes[k] === statusCode) ||
    "UNKNOWN_ERROR";
``` 

But it could be a good thing to transform this `StatusCode` object into an `enum` and use this enum everywhere. WDYT ?

Reply:
if it doesn't break anything else, I would rather see `LOCKED_DEVICE` than `0x5515` but keep in mind that some errors will share codes. Like maybe user refused, and something else, so the value can't be exclusive.